### DISCUSSION TITLE
Get to device page as a tap action

### DISCUSSION THREAD

**Discussion by mbo18** [home-assistant/frontend#20816] [Dashboard] [answered]
It would be nice if we could be able to set the tap action of a card (tile for example) to open the device page (whitout having the use URL and copy/paste the device URL). 
Then on a tap on a card I get directly redirected to the device the entity belongs to

**Comment by zaxxon72** [comment]
Yes, this would be very nice!

**Comment by karwosts** [comment, marked as answer]
I think this exists:

<img width="1118" height="991" alt="image" src="https://github.com/user-attachments/assets/da1615d8-8f61-4d86-909a-df5919b3d7e4" />

**Comment by mbo18** [comment]
Ah nice ! Thanks

**Comment by zaxxon72** [comment]
awesome!
